src/main/java/Duke.java

```diff
@@ -1,10 +1,65 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static void printList(Task[] list, int counter){
+        for(int i = 0; i < counter; i++){
+            System.out.print(i + 1 + ".");
+            printTask(list[i]);
+        }
+    }
+    public static void printTask(Task t){
+       System.out.println("[" + t.getStatusIcon()+ "] " + t.taskName);
+    }
+
+    public static int getTaskNumberArgument(String input){
+        int spaceIndex = input.indexOf(" ");
+        String taskNum = input.substring(spaceIndex + 1);
+        System.out.println(taskNum);
+        return Integer.parseInt(taskNum.trim());
+    }
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?");
+
+        Task[] list=new Task[100];
+        int taskCounter = 0;
+
+        String input;
+        Scanner sc = new Scanner(System.in);
+        input = sc.nextLine();
+
+        while(!input.equalsIgnoreCase("bye")){
+            // print list
+            if (input.equalsIgnoreCase("list")){
+                printList(list,taskCounter);
+            }
+            else if(input.startsWith("mark")){
```

Comment:
You can also the `else if` clause on the same line as the closing brace of the preceding `if/else` statement.

Reply:
Thanks for letting me know!